src/components/atoms/Timeline/index.tsx

```diff
@@ -35,15 +52,76 @@ const Timeline: React.FC<Props> = memo(
       strongScaleHours,
       currentPosition,
       events,
+      player: {
+        playSpeed,
+        onChangePlaySpeed,
+        formattedCurrentTime,
+        isPlayed,
+        isPlayBacked,
+        toggleIsPlayed,
+        toggleIsPlayBacked,
```

Comment:
Are `isPlayBacked` and `toggleIsPlayBacked` about playing in reverse order? If so, perhaps its better to be `isPlaying`, `isPlayingReversed`, `toggleIsPlaying` and `toggleIsPlayingReversed`.
If you can think of a better way to name them, or I am not quite right about what these mean, please let me know!

Reply:
This is fixed by [chore: rename playing boolean vars](https://github.com/reearth/reearth-web/pull/277/commits/7e73fb2aa13090f87552bcaaa04fe4d00e5e6969)